src/screens/Media/Media.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Suspense, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router';
+
+import useMedia from '#src/hooks/useMedia';
+import ErrorPage from '#src/components/ErrorPage/ErrorPage';
+import { getCustomScreen } from '#src/utils/common';
+import { mediaScreens } from '#src/customScreens';
+import Loading from '#src/screens/Loading/Loading';
+
+const Media = () => {
+  const params = useParams();
+  const id = params.id || '';
+  const { isLoading, error, data: item } = useMedia(id);
+  const { t } = useTranslation('error');
+  const MediaScreen = useMemo(() => getCustomScreen(mediaScreens, item), [item]);
+
+  if (isLoading) {
+    return <Loading />;
+  }
+
+  if (error || !item) {
+    return <ErrorPage title={t('video_not_found')} />;
+  }
+
+  return (
+    <Suspense fallback={<Loading />}>
```

Comment:
Does this work without Relay?

Reply:
Yes, it even is required to use suspense according to the docs: https://reactjs.org/docs/code-splitting.html#reactlazy

Reply:
Ah it goes with the lazy calls.  Got it.  Cool stuff!